sdk/keyvault/Azure.Security.KeyVault.Keys/src/EcKeyCreateOptions.cs

```diff
@@ -14,7 +14,7 @@ public class EcKeyCreateOptions : KeyCreateOptions
         public KeyCurveName Curve { get; set; }
         public bool Hsm { get; private set; }
 
-        public EcKeyCreateOptions(string name, bool hsm)
+        public EcKeyCreateOptions(string name, bool hsm, KeyCurveName curveName = default)
```

Comment:
Not really related to your change, but should we call that `bool useHardwareSecurityModule` or similar?

Reply:
We could, all languages went with with `hsm` and REST API documentation treats it like that. 
@schaabs  what do you think?